bigquery/src/main/java/org/apache/zeppelin/bigquery/BigQueryInterpreter.java

```diff
@@ -275,6 +278,9 @@ public static Iterator<GetQueryResultsResponse> run(final String queryString,
       GetQueryResults getRequest = service.jobs().getQueryResults(
           projectId,
           jobId);
+      if (region != null) {
```

Comment:
This check should be better. Not that somehow an empty string or a string with white spaces arrives here. Please consider the import of the function. It is best to also include `commons-lang3` in the pom.xml.
Sorry about the back and forth.
```suggestion
      if (StringUtils.isNotBlank(region)) {
```

Reply:
Done